dnngior/reaction_class.py

```diff
@@ -160,23 +162,8 @@ def __get_reactions_from_biochem_input(self, biochem_input):
                 reactions[reaction_id]['metabolites'] = {i:mets[i] for i in mets}
             else:
                 reactions[reaction] = {'lower_bound':-1.0, 'upper_bound':1.0}
-                reaction_split = react_d[reaction].split(' ')
-                mets = {}
-                side = 1
-                for i in reaction_split:
-                    if(len(i) > 1):
-                        stoc = 1.0
-                        if not('_' in i):
-                            if i[0].isdigit():
-                                stoc = float(i) * side
-                            else:
-                                if i == '<->':
-                                    side = -1
-                        else:
-                            mets[i] = stoc
+                reactions[reaction]['metabolites'] = eval(react_d[reaction][0])
```

Comment:
why using `eval` here? if there is a chance for an error, would not be better to have some catches for them ?